### DISCUSSION TITLE
create custom animations

### DISCUSSION THREAD

**Discussion by MonaAghili** [tailwindlabs/tailwindcss#14206] [Help]
I want to use this animation in the link below, but for some reason, I can't define it in my `styles.css` or `tailwind.config` files. Is it possible to create custom animations directly in HTML?

https://play.tailwindcss.com/3K55ujnfyy

**Comment by wongjn** [comment]
If by "create custom animations directly in HTML" you mean using an arbitrary value class, no. [You can apply the `animation` property with an arbitrary value class](https://tailwindcss.com/docs/animation#arbitrary-values) but the keyframes would need to be [defined in the Tailwind configuration](https://tailwindcss.com/docs/animation#customizing-your-theme) or CSS.

**Reply by MonaAghili** [reply]
We don't want to change config file.
so it's impossible to create `@keyframs` inside html class?
You know add animation like in what we have in gradient utilities `from-indigo-500 via-purple-500 to-pink-500`

**Reply by adamwathan** [reply]
Correct, I’d love to figure out a way to do this but haven’t come up with a good solution yet because keyframes have to be defined globally in CSS.

**Reply by MonaAghili** [reply]
Thank you @adamwathan for all your efforts. I hope you reach a good solution as soon as possible. With this feature, many projects that use Tailwind in different languages like Elixir, JavaScript, will move towards zero configuration and are experiencing a highly automated process.
